query/src/main/scala/filodb/query/exec/RangeVectorTransformer.scala

```diff
@@ -340,11 +340,9 @@ final case class RepeatTransformer(startMs: Long, stepMs: Long, endMs: Long, exe
             limit: Int,
             sourceSchema: ResultSchema,
             paramResponse: Seq[Observable[ScalarRangeVector]]): Observable[RepeatValueVector] = {
+    val rs = new RecordSchema(sourceSchema.columns, brSchema = sourceSchema.brSchemas)
```

Comment:
Instantiated once to avoid repeated instantiation of this same times as the number of range vectors.